### PR TITLE
Update snap binary names to snaptel/snapteld

### PR DESCRIPTION
**Pull request by mkleina**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,16 +80,16 @@ $ cd examples/tasks && ./run-mock-load.sh
 Script will start docker container in which it will download load collector, passthru processor and file publisher.
 Then it will start Snap daemon, load all plugins and start example [task](examples/tasks/task-load.json).
 
-Type `snapctl task list` to get the list of active tasks
+Type `snaptel task list` to get the list of active tasks
 ```
-bash-4.3# snapctl task list
+bash-4.3# snaptel task list
```
Comment (lmroz) [inline README.md]: I can live with that, but we use `$ snaptel` everywhere else.

File: README.md [hunk 2]
```diff
@@ -80,16 +80,16 @@ $ cd examples/tasks && ./run-mock-load.sh
...
 ID                                       NAME                                            STATE           HIT     MISS    FAIL    CREATED                 LAST FAILURE
 95b9fd8b-42d8-4836-be08-3865ba1f7926     Task-95b9fd8b-42d8-4836-be08-3865ba1f7926       Running         146     0       0       11:44AM 11-17-2016
 ```
 
-See realtime output from `snapctl task watch <task_id>` (CTRL+C to exit)
+See realtime output from `snaptel task watch <task_id>` (CTRL+C to exit)
 ```
-bash-4.3# snapctl task watch 95b9fd8b-42d8-4836-be08-3865ba1f7926
+bash-4.3# snaptel task watch 95b9fd8b-42d8-4836-be08-3865ba1f7926
```
Comment (lmroz) [inline README.md]: `bash-4.3#`

File: README.md [hunk 3]
```diff
@@ -99,7 +99,7 @@ NAMESPACE                                DATA    TIMESTAMP
 
 Stop task:
 ```
-$ $SNAP_PATH/bin/snapctl task stop 02dd7ff4-8106-47e9-8b86-70067cd0a850
+$ $SNAP_PATH/bin/snaptel task stop 02dd7ff4-8106-47e9-8b86-70067cd0a850
```
Comment (lmroz) [inline README.md]: $SNAP_PATH

Inconsistency with convention used above `bash-4.3#` vs `$`

Comment (mkleina) [inline README.md]: Ok, I just changed previous command prompt lines to "$" convention.

**Comment by lmroz** [comment]
LGTM

**Pull request merged**